### PR TITLE
track tool depencies

### PR DESCRIPTION
**Pull request by C0rby**
Added tracking for tool dependencies to be able to run `go mod tidy` without losing them.
More information: https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module

### CODE REVIEW

File: tools.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+package tools
+
+import (
+	"github.com/UnnoTed/fileb0x"
+	"github.com/cespare/reflex"
+	"github.com/haya14busa/goverage"
+	"github.com/kballard/go-shellquote"
+	"github.com/ogier/pflag"
+	"github.com/restic/calens"
```
Comment (refs) [inline tools.go]: I think these are intended to be blank imports. Also, I think this is meant for go-tools (https://github.com/golang/tools) not sure if it is intended to work with any package, as per the thread reads. Need to do more digging imho

Comment (refs) [reply tools.go]: I also see more benefits when it comes with a tool that generates code, as different versions of the tool will yield different results

Comment (C0rby) [reply tools.go]: I see your points but I find it a bit anoying that my tooling (vscode, make) is removing those dependencies from the go.mod file and I have to put them back manually.
Maybe I need to change something in my tooling or I'm the only one anoyed by that.

If it is like that I could also live with just closing this PR.

Comment (refs) [reply tools.go]: I'm okay with it :)

File: tools.go [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// +build tools
+
+package tools
+
+import (
+	_ "github.com/UnnoTed/fileb0x"
```
Comment (ownclouders) [inline tools.go]: Codacy Issue found: [a blank import should be only in a main or test package, or have a comment justifying it](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=4957233)

File: tools.go [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// +build tools
+
+package main
+
+import (
+	_ "github.com/UnnoTed/fileb0x"
+	_ "github.com/cespare/reflex"
+	_ "github.com/haya14busa/goverage"
+	_ "github.com/kballard/go-shellquote"
```
Comment (tboerger) [inline tools.go]: This is an indirect dependency, this would be added automatically

File: tools.go [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+	_ "github.com/ogier/pflag"
```
Comment (tboerger) [inline tools.go]: Same here

**Pull request merged**